models/remote_provider.go

```diff
@@ -95,7 +95,7 @@ func (l *RemoteProvider) loadCapabilities(token string) {
 	}
 
 	req, _ := http.NewRequest(http.MethodGet, remoteProviderURL.String(), nil)
-
+	// logrus.Debug(remoteProviderURL.String())
```

Comment:
remove this line.

Reply:
@Abhishek-kumar09 will you commit these corrections directly to this branch? You can use CTL+G on Github's web ui to do so.